### PR TITLE
Sub/feature/54 add edit testsuite form

### PR DESCRIPTION
**Pull request by tanmaythole**
Add New or Edit Testsuite form

**Summary**
This PR adds the functionality to add new or edit testsuite components. 

**Ticket Link**
https://trello.com/c/kcSmT8hR/54-creation-of-testsuite-frontend

### CODE REVIEW

File: src/Components/ProjectsComponent/TestsuiteComponent/AddNewTestsuite.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    onChange(e.target.name, e.target.value);
+  };
+
+  return (
+    <Form className="w-100">
+      <ViewComponent
+        title={formTitle(cat)}
+        type="save"
+        onSave={onSubmit}
+        onSaveDisabled={name.length === 0 || testcases.length === 0}
```
Comment (ChinmaySonawane) [inline src/Components/ProjectsComponent/TestsuiteComponent/AddNewTestsuite.jsx]: use correct naming convention for boolean

File: src/containers/ProjectContainer/MemberContainer/index.jsx [hunk 2]
```diff
@@ -1,136 +1,147 @@
...
+  }, [allUsers]);
+
+  const removeMember = (id) => {
+    dispatch(
+      removeMembersInProjectRequest({ project: projectName, members: [id] })
+    );
+  };
+
+  const onchangeMembers = (_name, val) => {
+    setAddMemberFormData((p) => ({
```
Comment (ChinmaySonawane) [inline src/containers/ProjectContainer/MemberContainer/index.jsx]: use `previousState` or `prevState`

**Review by ChinmaySonawane — APPROVED** [review]

File: src/Components/ProjectsComponent/TestsuiteComponent/AddNewTestsuite.jsx (hunk 1 above)
Comment (tanmaythole) [reply src/Components/ProjectsComponent/TestsuiteComponent/AddNewTestsuite.jsx]: will do it in separate PR

File: src/containers/ProjectContainer/MemberContainer/index.jsx (hunk 2 above)
Comment (tanmaythole) [reply src/containers/ProjectContainer/MemberContainer/index.jsx]: done

**Pull request merged**